### PR TITLE
feat(base-cluster/monitoring): monitor all namespaces by default

### PR DESCRIPTION
**Pull request by cwrau**
also add flag to only monitor rolled out namespaces

### CODE REVIEW

File: charts/base-cluster/templates/monitoring/kube-prometheus-stack/_prometheus_config.yaml [hunk 1]
```diff
@@ -43,11 +43,17 @@ prometheusSpec:
   probeSelector:
     matchLabels: {{- .Values.monitoring.labels | toYaml | nindent 6 }}
   additionalAlertRelabelConfigs:
+    {{- if .Values.monitoring.monitorAllNamespaces }}
+      {{- $namespaces := list .Release.Namespace "kube-node-lease" "kube-public" "kube-system" -}}
```
Comment (marvinWolff) [inline charts/base-cluster/templates/monitoring/kube-prometheus-stack/_prometheus_config.yaml:47]: So you can just monitor all or none?  
Or did i miss something?

Comment (marvinWolff) [reply charts/base-cluster/templates/monitoring/kube-prometheus-stack/_prometheus_config.yaml:47]: I think a seperation for the three namespaces manually added to the list and all other created ones would be good. Or maybe the possibility to simply pass a list,  overwriting the flag.

**Review by marvinWolff — APPROVED** [review]

File: charts/base-cluster/templates/monitoring/kube-prometheus-stack/_prometheus_config.yaml (hunk 1 above)
Comment (cwrau) [reply charts/base-cluster/templates/monitoring/kube-prometheus-stack/_prometheus_config.yaml:47]: Whoopsie, forgot the `not`

**Review by tasches — APPROVED** [review]
reviewed and templated. Looks good to me

**Pull request merged**